clang/include/clang/Driver/Action.h

```diff
@@ -76,9 +76,10 @@ class Action {
     StaticLibJobClass,
     BinaryAnalyzeJobClass,
     BinaryTranslatorJobClass,
+    BinaryModifyJobClass,
```

Comment:
Probably better to align this with the specific tool you're running:
```suggestion
    ObjcopyJobClass,
```

That is consistent with most of the other toolchain jobs (e..g. LipoJobClass, DsymutilJobClass, LinkerWrapperJobClass, etc).